pages/index.vue

```diff
@@ -74,20 +75,26 @@
                 <text-button to="/sponsor">{{ $t('sponsorUs') }}</text-button>
             </div>
         </i18n-page-wrapper>
-        <sponsor-modal v-model="isOpened" :context="selectedSponsor" />-->
+        <sponsor-modal v-model="isOpened" :context="selectedSponsor" />
```

Comment:
actually I just uncommented the existing sponsorship cards (plz check this [commit](https://github.com/pycontw/pycontw-frontend/blob/4f854d7f92c5dc4e3192d00c2ab373688dbf1916/pages/index.vue#L49-L77) for details)